libunwind/src/UnwindRegistersRestore.S

```diff
@@ -680,7 +680,7 @@ DEFINE_LIBUNWIND_FUNCTION(__libunwind_Registers_arm64_jumpto)
   ldr    x16,     [x0, #0x0F8]
   ldp    x0, x1,  [x0, #0x000]  // restore x0,x1
   mov    sp,x16                 // restore sp
-  ret    x30                    // jump to pc
```

Comment:
If the application is compiled with BTI then `br x30` will cause an exception as it jumps to a non landing pad instruction.
(We only generate landing pads for `return-twice` locations but a normal `catch` block won't get it)

Reply:
Yes you're right, and actually the way I've adjusted __unw_phase2_resume also means we need to change the BTI instruction in __libunwind_Registers_arm64_jumpto to BTI JC. I'll fix this.

Reply:
https://github.com/llvm/llvm-project/pull/102322 should fix this